service-config/src/main/java/com/palantir/config/service/ServiceConfiguration.java

```diff
@@ -44,22 +44,10 @@
      */
     public abstract List<String> uris();
 
-    // hides implementation details
     public static Builder builder() {
-        return ImmutableServiceConfiguration.builder();
+        return new Builder();
     }
 
-    // hides implementation details
-    public interface Builder {
-
-        Builder apiToken(BearerToken apiToken);
-
-        Builder security(SslConfiguration security);
-
-        Builder uris(Iterable<String> uris);
-
-        Builder from(ServiceConfiguration otherConfig);
-
-        ServiceConfiguration build();
+    public static final class Builder extends ImmutableServiceConfiguration.Builder {
```

Comment:
This means a lot of the generated class methods will be accessible/viewable
to the calling user (like fromAttributes, withName, withTags) and will NOT
include convenience .of() methods that we've specifically written in many
places in foundry